src/common/common.csproj

```diff
@@ -46,31 +46,24 @@
   </ItemGroup>
 
   <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />

```

Comment:
@Eneuman Bridge uses both self contained for VS extensions and framework dependent executables for VS code extensions. does this change affect self contained builds ? Did you try running ```dotnet publish -r RID --self contained true``` ?

Reply:
@Eneuman I tried myself and compared your changes with the one in the main branch without shared framework. seems like some of the packages are missing ex: AspNetCore.JsonPatch refer below for a screenshots. 

Microsoft.Extensions.DependencyModel.dll
![image](https://user-images.githubusercontent.com/105889062/228290174-cd312896-83aa-404f-89c0-ab631c703526.png)

Microsoft.Dotnet.PlatformAbstractions.dll
![image](https://user-images.githubusercontent.com/105889062/228290225-1831d43f-13b8-4c65-a750-55941fb212b6.png)

Microsoft.CodeAnalysis.dll
![image](https://user-images.githubusercontent.com/105889062/228290282-00982201-9e50-4fe9-b49e-669e791fb1c4.png)

Microsoft.AspNetCore.JsonPatch.dll
![image](https://user-images.githubusercontent.com/105889062/228290378-10222702-5891-46d5-ab3f-7d19957971ce.png)

also System.Security.Permissions.dll is also missing. Any idea on these ?? 

Note: also using shared framework had both advantages and disadvantages,

Reply:
@hsubramanianaks yes, jsonpatch was removed in a earlier PR (when Newtonsoft was replaced with STJ) and the other packages are removed by the publish process because it can not find any references to them so I think it's all good. I didn't know B2K used self contained builds so I'm gonna do some extra testing around that just to make sure. 

What parts are running as self-contained?

Reply:
@Eneuman Thanks, I will send this to the testing team and see how it goes. Yes PublishReadyToRun=true seems like good option. I can do another PR on the pipelines to include them. Thanks for the suggestion. :)